lib/destack/src/lib/css/index.ts

```diff
@@ -18,9 +18,10 @@ const appendTailwindCss = (newEditor, setCssLoaded) => {
 }
 
 const appendCustomCss = () => {
-    document.querySelector("html").style.height = '100%'
-    document.querySelector("body").style.height = '100%'
-    document.querySelector("#__next").style.height = '100%'
+    document.querySelector("html")!.style.height = '100%'
+    document.querySelector("body")!.style.height= '100%'
+    let next: HTMLElement = document.querySelector("#__next")!
```

Comment:
Did you encounter any problems with?
```ts
document.querySelector("#__next")!.style.height = '100%'
```

I'm thinking the variable here can be a`const`:
`let next: HTMLElement ` → `const next: HTMLElement `

Reply:
The problem is that  the normal html elements (such as body, div) have the style property, but others don't.

There probably is a way to do this cleanly, I'll look into this.

Reply:
@LiveDuo t

> const next: HTMLElement
this has been fixed in another commit I think